updater.go

```diff
@@ -263,6 +334,62 @@ func (u *Updater) NeedUpdate(ctx Context) (upToDate bool, err error) {
 	return update.NeedUpdate, nil
 }
 
+func (u *Updater) CheckAndDownload(ctx Context) (availableAndDownloaded bool, err error) {
```

Comment:
maybe this can be split up into two bools, `available` and `downloaded` the RPC should only expect `available=true` but `downloaded` can be helpful for testing the cache behavior in tests

Reply:
@joshblum I changed this so that the second return value is `updateCached` which is true if the update is already on disk **or** if it's just been downloaded.

Reply:
isn't this equivalent to `updateAvailable`?